### PR TITLE
Implement support for PHP CS ignore option

### PR DESCRIPTION
**Pull request by stmswitcher**
Hey,

I've been using php-git-hooks in a while in some of the projects, thank you for this great solution which makes it easier to keep code nice and clean!

However, some of the projects using either outdated code base, or at least contain some "heavy" parts which are always a topic for refactoring.
Recently I've been missing the configuration option to pass ignore patterns to PHPCS to specify the path for files which doesn't have to be checked and I wasn't able to find a way to do it with php-git-hooks, even if PHPCS supports the `--ignore` option.

So, I've implemented a feature to be able to use that option with php-git-hooks.
Not sure if there was a way to do it with IgnoreFiles, so it's made as it is right now.

P.S. Probably, the Ignore class should be named differently, but I had no better idea at the moment :)

### CODE REVIEW

**Comment by bruli** [comment]
Sorry but I think you must use phpcs annotations to ignore your code. Is better solutions than this PR.

You can see it, https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage

**Comment by stmswitcher** [comment]
@bruli, yes, that's how we currently do it.
Still I'd prefer to keep files clean of annotations and not to bother with them when there's another new file which won't be accepted by CodeSniffer anyway. And since all these files located under the same path, I'd rather ignore this one.

Also if, for example, git hooks would be implemented for the project with a lots of legacy code, devs will have to add annotations to many files. And with this option, it could be done with a single line.

**Comment by bruli** [comment]
Ok, I will review the PR.

But, could you review travis conflicts?.

**Comment by stmswitcher** [comment]
Yes, I'm currently on it.

**Comment by stmswitcher** [comment]
@bruli, done.

**Comment by bruli** [comment]
Hi.
I'm back from my holidays.
I'm reviewing your code and I have some sugestions. Please, check comments in earch file.

File: php-git-hooks.yml.sample [hunk 1]
```diff
@@ -23,6 +23,7 @@ pre-commit:
     phpcs:
         enabled:     true
         standard:    PSR2
+        ignore:      '*/migrations/*'
```
Comment (bruli) [inline php-git-hooks.yml.sample]: When tool run from "composer install", don't ask about ignore option. It must ask.

File: src/PhpGitHooks/Module/Configuration/Domain/PhpCs.php [hunk 2]
```diff
@@ -58,14 +64,23 @@ public function getStandard()
...
      * @return ToolInterface
      */
     public function setEnabled(Enabled $enabled)
     {
         return new self(
             new Undefined(false),
             $enabled,
-            new PhpCsStandard(null)
+            new PhpCsStandard(null),
+            new Ignore('')
```
Comment (bruli) [inline src/PhpGitHooks/Module/Configuration/Domain/PhpCs.php]: '' value is not a good string. Must replace it by null.

**Comment by stmswitcher** [comment]
Hi!
Thanks for a feedback :)
I've updated the code.

**Comment by bruli** [comment]
Merged.

Thanks.

**Comment by stmswitcher** [comment]
Thank you!

**Pull request merged**